### PR TITLE
[many ports] Updates 2019.05.19

### PR DESCRIPTION
**Pull request by ras0219-msft**

### CODE REVIEW

File: ports/libbson/portfile.cmake [hunk 1]
```diff
@@ -3,8 +3,8 @@ include(vcpkg_common_functions)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO mongodb/mongo-c-driver
```
Comment (cenit) [inline ports/libbson/portfile.cmake]: Is this library a clone of mongo-c-driver?? Same repo, same ref...

Comment (ras0219-msft) [reply ports/libbson/portfile.cmake]: Both libraries are built from the same source repo, but you can build the binaries independently. Notice the `-DENABLE_BSON=ON` and `-DENABLE_MONGOC=OFF`.

**Comment by cenit** [comment]
this port broke theia on linux... how is it a green check? (I'm just curious, I'd like to be able to do it for my own PR 🤣 )
anyway, I'll try my best to restore it in #6730

**Pull request merged**